source/draw/gpu/opengl/OpenGLCanvas.ooc

```diff
@@ -28,6 +28,38 @@ OpenGLCanvas: class extends OpenGLSurface {
 	_target: OpenGLPacked
 	_renderTarget: GLFramebufferObject
 	context ::= this _context as OpenGLContext
+	draw: override func ~DrawState (drawState: DrawState) {
+		// Set shader
```

Comment:
If you want these comments, change this one to `//Set shader, viewport, and transform` and remove those on line 34 and 37.